package.json

```diff
@@ -17,7 +17,8 @@
     "lerna:publish": "lerna publish --force-publish --conventional-commits --no-changelog --exact",
     "lint": "prettier --check . && eslint .",
     "lint:fix": "prettier --write .",
-    "link:prepare": "lerna exec -- node ../../../tools/silent.js yarn link --link-folder ../../../.links --silent --no-bin-links",
+    "link:prepare": "lerna exec -- yarn link --silent --no-bin-links --link-folder ../../../.links",
```

Comment:
The old command was failing in Windows, and after trying multiple combos, this was the only one that seemed to work.